lib/jsonapi/utils/string.ex

```diff
@@ -122,19 +79,25 @@ defmodule JSONAPI.Utils.String do
       iex> camelize("top_posts")
       "topPosts"
 
+      iex> camelize(:top_posts)
+      :topPosts
+
       iex> camelize("_top_posts")
       "_topPosts"
 
       iex> camelize("_top__posts_")
       "_top__posts_"
 
   """
+  @spec camelize(atom) :: atom
   def camelize(value) when is_atom(value) do
     value
     |> to_string()
     |> camelize()
+    |> String.to_atom()
```

Comment:
For outgoing data, what reason would we convert it back to an atom?  `underscore` did this as well

Reply:
IIRC this came from some code review feedback here: https://github.com/jeregrine/jsonapi/pull/149#discussion_r243855630

Reply:
I think we can ditch the `to_atom/1` calls, I'm not sure that's a good habit to encourage.